filter/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>no.sb1.troxy</groupId>
```

Comment:
@sconrads, skal det være "sparebank1" istedet for "sb1" her?

Reply:
Ja, det skal være no.sparebank1
Noe annet vil ikke Maven Central godta